lib/modules/webserver/server.js

```diff
@@ -77,14 +83,16 @@ class Server {
       if (err) {
         return callback(err);
       }
-      const msg = (
-        __('webserver available at') + ' ' +
-          ('http://' + canonicalHost(self.hostname) + ':' + self.port).bold.underline.green
-      );
-      callback(null, msg, self.port);
+
+      callback(null, self._getMessage(), self.port);
     });
   }
 
+  _getMessage() {
+    return __('webserver available at') + ' ' +
+    ('http://' + canonicalHost(this.hostname) + ':' + this.port).bold.underline.green;
```

Comment:
Probably a good opportunity to use ES' template variable syntax:
```
`${__('webserver available at')} http://${canonicalHost(this.hostname)}:${this.port)}`.bold.underline.green;
```

Also something to keep in mind, if I remember correctly there's `https` support in the making. We probably want to update it here for consistency then as well.